applications/nrf_desktop/src/modules/usb_state.c

```diff
@@ -962,6 +979,68 @@ static int usb_init_next_hids_init(void)
 	return err;
 }
 
+static int usb_thread_req_send(bool enable)
+{
+	uint32_t events;
+
+	__ASSERT_NO_MSG(!k_event_test(&usb_event, USB_REQ_ENABLE | USB_REQ_DISABLE));
+
+	LOG_DBG("%sabling USB", enable ? "En" : "Dis");
+	(void) k_event_set(&usb_event, enable ? USB_REQ_ENABLE : USB_REQ_DISABLE);
+	events = k_event_wait(&usb_event,
+			      USB_RSP_SUCCESS | USB_RSP_FAIL,
+			      false,
+			      K_MSEC(USB_REQ_TIMEOUT_MS));
+	(void) k_event_clear(&usb_event, events);
+
+	if (!events) {
```

Comment:
Why not `if ... else if ... else` with `err` local variable ? You could avoid early returns and assertion at the end of this function